recipe/meta.yaml

```
@@ -10,26 +10,28 @@ source:
     sha256: 643f3b84ac511b301e481c35278e6a4deacb9e945ad1e36dc3ff4b5f45300b39
 
 build:
-  number: 0
-  skip: true  # [not win64]
+  number: 1
   run_exports:
-    - {{ pin_subpackage(name, exact=True) }}
+    - {{ pin_subpackage(name, max_pin='x.x.x') }}
 
 requirements:
   build:
-    - vs2017_win-64          # [win64]
+    - {{ compiler('c') }}
```

Both compilers are needed.
```suggestion
    - {{ compiler('c') }}
    - {{ compiler('cxx') }}
```